### PR TITLE
SCAL-81383-typedoc-url-changes-based-on-deployment-type

### PR DESCRIPTION
**Pull request by pankajp-ts**
Ticket - https://thoughtspot.atlassian.net/browse/SCAL-81383

### CODE REVIEW

File: docs/src/utils/app-utils.ts [hunk 1]
```diff
@@ -46,12 +49,27 @@ export const queryStringParser = (queryParamStr: string) => {
...
     navPrefix += TS_PAGE_ID_PARAM;
     queryParamObj[NAV_PREFIX] = navPrefix;
+
+    // prepare and set 'Preview in Playground' button URL prefix
     queryParamObj[
         PREVIEW_PREFIX
     ] = `${tsHostUrl}/#${queryParamObj[TS_APP_ROOT_PARAM]}`;
 
+    // set deployed environment of 'Visual Embed SDK'
+    const buildEnv = process.env.BUILD_ENV || 'LOCAL';
```
Comment (ayonghosh) [inline docs/src/utils/app-utils.ts:62]: Use a constant for LOCAL

File: docs/src/utils/app-utils.ts [hunk 2]
```diff
@@ -46,12 +49,27 @@ export const queryStringParser = (queryParamStr: string) => {
...
+    let deployedEnv = '';
+    if (buildEnv !== 'LOCAL') {
+        deployedEnv = `${buildEnv === 'PROD' ? 'release' : 'dev'}/`;
```
Comment (ayonghosh) [inline docs/src/utils/app-utils.ts:65]: Use constants for these strings

**Review by ashubham — APPROVED** [review]

File: docs/src/utils/app-utils.ts (hunk 2 above)
Comment (pankajp-ts) [reply docs/src/utils/app-utils.ts:65]: done

File: docs/src/utils/app-utils.ts (hunk 1 above)
Comment (pankajp-ts) [reply docs/src/utils/app-utils.ts:62]: done

**Review by ayonghosh — APPROVED** [review]

**Pull request merged**